JetNews/app/src/main/java/com/example/jetnews/ui/Vectors.kt

```diff
@@ -31,7 +31,9 @@ import androidx.ui.res.vectorResource
 fun VectorImageButton(@DrawableRes id: Int, onClick: () -> Unit) {
     Ripple(bounded = false) {
         Clickable(onClick = onClick) {
-            VectorImage(id)
+            Container(expanded = true) {
```

Comment:
This changes the rendering of the app – the icon is pushed to the right.

Reply:
I don't get visual difference, the icon is placed exactly on same location, it's just clickable properly in entire area.

I get this:
![image](https://user-images.githubusercontent.com/3145438/68324245-4ab3a300-00c7-11ea-9ab9-ba8013cba131.png)

Reply:
True, with this fix the icon is pushed a bit to right, but actually it's correctly placed to center of `navigationIcon` in the `TopAppBar`, because without originally it was aligned to left side of it.